.circleci/config.yml

```diff
@@ -66,6 +76,119 @@ jobs:
       - run:
           name: Push Docker image to DockerHub
           command: docker push openmf/ph-ee-bulk-processor:latest        
+      - run: |
+          kubectl config use-context arn:aws:eks:$REGION:419830066942:cluster/sit
```

Comment:
can we use this '419830066942' also from the circle CI env variable just like $REGION?
this is also a sensitive information